docs/development.md

```diff
@@ -1,29 +1,18 @@
-# Local Development
+# Faasm Development
 
 This guide is only relevant for those wanting to dig deeper or make changes to
-Faasm itself.  If you'd just like to write and run functions, you can refer to
+Faasm itself. If you'd just like to write and run functions, you can refer to
 the [set-up](setup.md) instructions. 
 
 ## Recommended Set-up
 
 The Faasm codebase is all standard C/C++ and Python, so should work on a range 
-of set-ups, however, the only one that is recommended and well tested is:
+of set-ups, however, the recommended configuration is:
 
-- Ubuntu 18.04
-- Python 3.7
-- Clang, not GCC
-
-All scripted CMake builds also use Ninja, so it's safest to use that too.
-
-This means your CMake commands will look something like:
-
-```
-cmake \
-    -GNinja \
-    -DCMAKE_CXX_COMPILER=/usr/bin/clang++-10 \
-    -DCMAKE_C_COMPILER=/usr/bin/clang-10 \
-    ...
-```
+- Ubuntu 20.04
+- Python 3.8
+- Clang 10
+- Ninja (rather than Make)
```

Comment:
Worth it to include `CMake` in the list of requirements? I think the packaged version in 20.04 is [already sufficiently good](https://cliutils.gitlab.io/modern-cmake/chapters/intro/installing.html) but maybe worth pointing out.